indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -502,7 +504,7 @@ private void initializeSubTaskCleaner()
     if (isParallelMode()) {
       currentSubTaskHolder = new CurrentSubTaskHolder((currentRunnerObject, taskConfig) -> {
         final ParallelIndexTaskRunner runner = (ParallelIndexTaskRunner) currentRunnerObject;
-        runner.stopGracefully();
+        runner.stopGracefully(null);
```

Comment:
Should we add some generic Cleaner stop reason here?

Reply:
I left this as is, I don't think it'd really add much and it would need to be checked elsewhere to see that it doesn't override the error message on task failure